### ISSUE TITLE
Improve login button

### ISSUE THREAD

**Issue by jainaman224** [algobook/algoBook#5]
Suggestions on login button at [home page](http://algobook.net/).

**Comment by RusinovAnton** [comment]
I'm taking up this one, alright?

**Comment by RusinovAnton** [comment]
I guess we might consider improvement on design of sign up/ sign in forms also?

**Comment by RusinovAnton** [comment]
LMK your thoughts on these

![image](https://user-images.githubusercontent.com/13165540/31057977-92dce594-a6f4-11e7-8aba-737792af7982.png)

![image](https://user-images.githubusercontent.com/13165540/31057970-7e7f5dfc-a6f4-11e7-9b28-3a3a93eb2881.png)

![image](https://user-images.githubusercontent.com/13165540/31057996-d695be5a-a6f4-11e7-985f-86126422ca33.png)

**Comment by RusinovAnton** [comment]
K, I got it, we should make it looks like on other pages:

![image](https://user-images.githubusercontent.com/13165540/31058407-ac5ce3e6-a6fb-11e7-895b-5c1c73e64c6f.png)

**Comment by poush** [comment]
@RusinovAnton Yeah feel to take this issue. and yes we need to work on improving signup and login forms as well.
